README.md

```diff
@@ -24,9 +24,17 @@ What is working :
 - trackpad
 - display brightness/ brightness control (F11 & F12)
 - battery managment
+- HDMI (Video and sound)
 
 Note : Sound on headphones is also working, please follow 'Headphones sound fix' down below. 
 
+Note 2: To HDMI work perfectly, it is necessary:
+- Go to Clover Configurator, go to Mount EFI and mount the your EFI partition;
+- Open the partition and open the file: /EFI/CLOVER/config.plist
+- In Clover, open Boot tab and in Boot Arguments panel, click with right mouse and add option: 
+
+`WhateverGreen/agdpmod=vit9696`
+- Save the alterations, restart your computer and check if worked.
```

Comment:
Please be careful about English faults when contributing.

Reply:
I will do another PR. I'm not a native English speaker, sorry about that.